src/scripts/farming/mutation/mutationTypes/GrowNearFlavorMutation.ts

```diff
@@ -48,4 +48,30 @@ class GrowNearFlavorMutation extends GrowNearMutation {
         return this.flavorReqs.every((value, idx) => value[0] <= nearFlavors[idx] && nearFlavors[idx] <= value[1]);
     }
 
+    /**
+     * Handles getting the mutation chance.
+     * Will decrease the mutation chance if the mutatedBerry already exists around this one.
+     * @param idx The plot index
+     */
+    mutationChance(idx: number): number {
+        const plots = Plot.findNearPlots(idx);
+        if (!plots.length) {
+            return 0;
+        }
```

Comment:
In the unlikely event this does happen, should it not just return the default mutation chance?
```suggestion
        if (!plots.length) {
            return super.mutationChance(idx);
        }
```